src/filetreediff.js

```diff
@@ -240,10 +215,72 @@ export class FileTreeDiffElement extends LitElement {
     this.docDiffEnabled = false;
   };
 
+  getChunks() {
+    const chunks = document.querySelectorAll(
+      ".doc-diff-added, .doc-diff-removed",
+    );
+    // These are the nodes we consider a "section for a chunk". The classes
+    // `.doc-diff-*` are added to the _word_ that changed, but we want to
+    // highlight the parent element of it, being a more important section of the
+    // page (i.e `sectionNodes`)
+    //
+    // Examples:
+    //
+    //   - If the class is added to a `span/ins/del` (word/sentence
+    //     deleted/added inside a paragraph), we will return its parent `p`.
+    //   - If the class is added to a `section`, we will return the same
+    //     `section` element.
+    //   - If the class is added to a `li`, we will return the same `ul`/`ol`
+    //     element.
+    const sectionNodes = [
+      "section",
+      "h1",
+      "h2",
+      "h3",
```

Comment:
```suggestion
      "h3",
      "h4",
      "h5",
      "h6",
```

Seems reasonable for completeness?